src/commands/index.ts

```diff
@@ -31,6 +31,7 @@ class ContentfulMdg extends Command {
     jsdoc: Flags.boolean({ char: 'd', description: 'add JSDoc comments' }),
     typeguard: Flags.boolean({ char: 'g', description: 'add type guards' }),
     response: Flags.boolean({ char: 'r', description: 'add response types' }),
+    modifier: Flags.string({ char: 'm', description: 'add default modifier (v10 only)' }),
```

Comment:
Should we:
- allow all possible modifiers
- allow an array of modifiers to also support combinations?